Test/cli/errorLimit.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %exits-with 4 %baredafny verify %args --error-limit=2 "%s" > "%t"
+// RUN: %exits-with 4 %baredafny verify %args --error-limit=0 "%s" > "%t"
```

Comment:
It  looks like this one was intended to test the error limit. Is it too non-deterministic ?

Reply:
Yeah, I originally created that one to test the `--error-limit` flag, and then found that limiting it to a subset of the errors was non-deterministic, so I'd use it in the mode where it raised the limit, instead. At first, I only had that fix on the Z3 4.12.1 test update branch, but it really belongs everywhere.